src/main/java/pl/javastart/task/Telewizor.java

```diff
@@ -0,0 +1,29 @@
+package pl.javastart.task;
+
+public class Telewizor {
+    private boolean onOff;
```

Comment:
Trzeba popracować nad nazwą zmiennej, na taką, która jednoznacznie wskazuje jaki jest stan. 
Przekładowo gdyby nazwa `isOn`, to `true` jednoznacznie wskazuje, że jest włączone.
Aktualnie bez obejrzenia pozostałej części kodu to trochę zgadywanka.
Nazwę możesz zmienić za pomocą Shift+F6